webapp/channels/src/components/browse_channels/browse_channels.tsx

```diff
@@ -29,7 +30,14 @@ import './browse_channels.scss';
 
 const CHANNELS_CHUNK_SIZE = 50;
 const CHANNELS_PER_PAGE = 50;
-const SEARCH_TIMEOUT_MILLISECONDS = 100;
+
+var displaySettings: DisplaySettings = {
+    CustomURLSchemes: [],
+    MaxMarkdownNodes: 0,
+    SuggestionDebounceDelay: 0,
+}
```

Comment:
why are we defining this?

Reply:
I defined that to initialize the `DisplaySettings` struct in a bid to access the `SuggestionDebounceDelay` field.

Reply:
I defined that to initialize the `DisplaySettings` struct in a bid to access the `SuggestionDebounceDelay` field.

Reply:
Hi, @M-ZubairAhmed. I have not gotten a response from you concerning this.

Reply:
Uhhh. @M-ZubairAhmed :)

Reply:
Apologies @TheInvincibleRalph this missed in the pool of notifications, let me get back to you today with this one

Reply:
this is incorrect you need to read the value from the config.